src/Hydrators/CarHydrator.php

```diff
@@ -14,4 +14,28 @@ public static function getCars(\PDO $db): array
         $query->setFetchMode(\PDO::FETCH_CLASS, CarEntity::class);
         return $query->fetchAll();
     }
+
+    public static function getCar(\PDO $db, $id): CarEntity
+    {
+        $query = $db->prepare('SELECT `make`, `model`, `year`,`color`,`location`,`image` FROM `cars` WHERE `id` = ?');
```

Comment:
There seems to be some inconsistencies in spaces in-between commas and back ticks e.g. there's a space before `model` and `year` but not before `colour` or `location` or `image`. Not sure if this is deliberate